entries/l/leetcode.com.json

```
@@ -0,0 +1,19 @@
+{
+    "LeetCode": {
+        "domain": "leetcode.com",
+        "additional-domains": [
+            "leetcode.cn"
+        ],
+        "contact": {
+            "facebook": "LeetCode",
+            "twitter": "LeetCode"
```

They advertise this email address on their Facebook page.
```suggestion
            "twitter": "LeetCode",
            "email": "admin@leetcode.com"
```